ui/temboardui/static/src/components/home/InstanceList.vue

```diff
@@ -17,16 +18,15 @@ const loading = ref(false);
 const instances = ref([]);
 const search = ref(route.query.q);
 const sort = ref(route.query.sort || "status");
-const groups = ref([]);
-const groupsFilter = ref(route.query.groups ? route.query.groups.split(",") : []);
+const groupsFilter = reactive(route.query.groups ? route.query.groups.split(",") : []);
```

Comment:
Can you please elaborate why you changed from `ref` to `reactive` here?

Reply:
Filter changes are not caught by the watch function used to update the route, in this case the reactive object must be used to correct the problem.

Reply:
Can you please isolate the change to "fix the groupsFilter" into its own commit? And explain what was "fixed".

I think there are other options than using `reactive`.

Reply:
Ou en commentaire du code, ça évite de blamer et de dépendre de git.

Reply:
Thanks for splitting commits and finding an other workaround.